### ISSUE TITLE
Inconsistent MFD binning on new Solution plots

### ISSUE THREAD

**Issue by chrisbc** [opensha/opensha#43]
After running a few Inversion Solution reports I noticed that some of the new **SolMFDPlot** plots look very different to their histogram equivalents later in the report which use the **RupHistogramPlots** class. Some of the difference is scale related, which is expected but other differences are more extreme.

I suspect this is at least partly due to histogram binning differences - opensha  MFD plots we're used to seeing have bin centres on 0.1 intervals, with a 0.05  offset, but this offset is missing in the new  **Solution** MFD plot . I've attached some examples below with their TXT data,  and  here are the supposedly equivalent data sections from those txt files.

~NB Matt G suggested that the old opensha MFD binning is not the typtical one. I suppose we can change the binning used to different offsets if that's what everyone prefers, but I assume they should be consistent across plots.~ EDIT: Sorry I misunderstood Matt. It seems that the old 0.05 offsets are the 'correct' ones.

```
DATASET #1

     Name: Unique
   Points: 35
     Info: 

Data[x,y]:
6.05	  0.0
...
6.95	  0.0
7.05	  0.049770657
7.15	  0.0
7.25	  0.030500453
7.35	  0.024023632
7.45	  0.018684246
7.55	  0.014782539
7.65	  0.011600159
7.75	  0.009120146
7.85	  0.0071507185
7.95	  0.0030206162
8.05	  5.8427907E-4
```

VS 

```
DATASET #3

     Name: Solution
   Points: 51
     Info: Arbitrary Incremental Magnitude Frequency Dististribution

Data[x,y]:
5.0	  0.0
...
7.0	  0.0
7.1	  0.049770657
7.2	  0.030500453
7.3	  0.019822307
7.4	  0.0060619186
7.5	  0.031394396
7.6	  0.0115745645
7.7	  0.005903618
7.8	  0.008851861
7.9	  0.003727054
8.0	  0.0014017088
8.1	  4.8354216E-4
```

**Plot examples**

![hist_MAG_rates_log](https://user-images.githubusercontent.com/8263318/130527895-8016ed47-e790-4ae5-965c-ac84243b7300.png)

![mfd_plot_Total_Target_MFDs](https://user-images.githubusercontent.com/8263318/130527820-dd0398c4-66c9-4b37-b639-e198f96bfc68.png)

**TXT examples**

[hist_MAG_rates_log.txt](https://github.com/opensha/opensha/files/7035271/hist_MAG_rates_log.txt)

[mfd_plot_Total_Target_MFDs.txt](https://github.com/opensha/opensha/files/7035269/mfd_plot_Total_Target_MFDs.txt)

**Comment by kevinmilner** [comment]
Oh yeah, good catch. Fix is easy, standby

**Comment by kevinmilner** [comment]
Fixed in 865e02c.

Fixing this also revealed another bug related to building the truly off-fault MFD on the modular branch, which I fixed in 3b6eb6f. If your code on the modular branch called InversionFaultSystemSolution.getFinalTrulyOffFaultMFD(...), any precomputed & serialized grid-source providers will be wrong.
